Tools/CISupport/ews-build/steps.py

```diff
@@ -2533,12 +2532,6 @@ def __init__(self, **kwargs):
         shell.Test.__init__(self, logEnviron=False, **kwargs)
         self.incorrectLayoutLines = []
 
-    def _get_patch(self):
```

Comment:
Is this dead code? Can you please do git blame to find/confirm which commit made this un-used?

Reply:
Searching through the file, it's dead code. We added it in https://github.com/WebKit/WebKit/commit/5e44cd05acec7, but we don't call `_get_patch()` in `RunWebKitTests` any more. We only invoke it in `ApplyPatch`, `CheckChangeRelevance` and `FindModifiedLayoutTests`

Reply:
ok, _get_patch() invocation was removed subsequently in https://github.com/WebKit/WebKit/commit/658f4640f112.